packages/language/system-test/language.js

```diff
@@ -331,7 +359,18 @@ describe('Language', function() {
         assert.deepEqual(annotation.tokens[0], {
           text: 'Hello',
           partOfSpeech: 'Other: foreign words, typos, abbreviations',
-          partOfSpeechTag: 'X'
+          tag: 'X',
+          aspect: 'ASPECT_UNKNOWN',
```

Comment:
Can we normalize the `0` case for these `{{type}}_UNKNOWN` and return `undefined` or `false`? That would help the user:

```js
detectSyntax(..., function(err, syntax) {
  // :(
  var genders = syntax.tokens.filter(function(token) {
    return token.gender !== 'GENDER_UNKNOWN'
  })

  // :)
  var genders = syntax.tokens.filter(function(token) {
    return token.gender
  })
})
```